.github/workflows/_publish-docs-on-release.yml

```diff
@@ -34,14 +44,14 @@ jobs:
           auto-update-conda: true
           environment-file: environment.yml
           auto-activate-base: false
-          python-version: 3.13
+          python-version: ${{ inputs.python_version }}
```

Comment:
For packages like pdffit2 and regolith, we still want to use this beautiful reusable workflow.